### ISSUE TITLE
Make keyboard shortcuts fully configurable

### ISSUE THREAD

**Issue by sedwards2009** [sedwards2009/extraterm#79]

**Comment by nbelakovski** [comment]
As a stopgap, perhaps add some text to the key bindings page pointing people to the JSON file and informing them that manual modifications to the file will take effect upon restart of extraterm? Not the same as fully configurable, but at least unblocks people.

Also, this is a separate issue, but I tried setting new tab and close tab to Ctrl+T and Ctrl+W respectively, but neither worked (on windows). I checked that no other binding was using those, but ultimately had to switch them to Ctrl+Shift+T/Ctrl+Shift+W. Any idea why that might be?

**Comment by sedwards2009** [comment]
Done in v0.37.0
